sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -2,12 +2,25 @@
 
 var srlog = SIREPO.srlog;
 var srdbg = SIREPO.srdbg;
-SIREPO.PLOTTING_LINE_CSV_EVENT = 'plottingLineoutCSV';
 SIREPO.DEFAULT_COLOR_MAP = 'viridis';
+SIREPO.PLOT_DIRECTIVES = ['heatmap', 'parameterPlot', 'particle', 'plot2d', 'plot3d',];
+SIREPO.PLOTTING_LINE_CSV_EVENT = 'plottingLineoutCSV';
 SIREPO.SCREEN_DIMS = ['x', 'y'];
 SIREPO.SCREEN_INFO = {x: { direction: 1 },  y: { direction: -1 }};
 
-SIREPO.app.factory('plotting', function(appState, frameCache, panelState, utilities, requestQueue, simulationQueue, $interval, $rootScope) {
+//const directivePlot2d = 'plot2d';
+
+angular.element(document).ready(function() {
+    for (let d of SIREPO.PLOT_DIRECTIVES) {
+        SIREPO.PANELS[d].addWatermark = function(authState) {
+            if (! authState.isPremiumUser()) {
```

Comment:
Let the server make this decision. There's no reason for the GUI to decide when or when not to watermark. 

Add something to authState called `watermarkReports` that gets decided on the server and is based on a feature_config flag (`watermark_reports`). We need to be able to turn this off on the server side, because enterprise installs will not want it, and they won't necessarily mark each user as premium. In `sirepo.auth._auth_state`, we would set `watermarkReports" to whether `watermark_reports` is true and whether the user is a premium user.

Reply:
072929c

I put ```watermark_reports``` in a feature config section (?) ```gui```, not sure about that though